test/integration/crcsuite/crcsuite.go

```diff
@@ -308,6 +310,20 @@ func StartCRCWithDefaultBundleAndHypervisorSucceedsOrFails(hypervisor string, ex
 	return err
 }
 
+func StartCRCWithDefaultBundleAndNameServerSucceedsOrFails(nameserver string, expected string) error {
+
+	var cmd string
+	var extraBundleArgs string
+
+	if !constants.BundleEmbedded() {
```

Comment:
@tsedmik this is not going to work and I think that is what @jsliacan is working on.

Reply:
@praveenkumar - yes, it won't work with embedded bundle. But this is how the rest of the code on the master is right now. I suggest this can go in as it won't make things worse. And a comprehensive solution is in #724. WDYT?